### DISCUSSION TITLE
Some RTMP streams won't start - why? Can I, can we fix it?

### DISCUSSION THREAD

**Discussion by Balls0fSteel** [tiangolo/nginx-rtmp-docker#82] [Questions]
I know this project has not received updates lately, but this is the most active related project still.

My question is: I used `libnginx-mod-rtmp` and I put my setup together by hand. (yeah, I know, copy-paste is hard job.) So yeah I am not using this exact repo, but I checked the Dockerfile and it's the same just a comfier solution.

Anyway. All is well. Sometimes.
But sometimes, a stream or two just won't start.

I read a lot on the topic and it seems the control message is not going through or something? I tried to debug it but I can't make heads or tails of rtmp's inner-workings. I also found various snippets, patches online but nothing helps it seems. Even worse, I test it and it just works. When I actually want to use it _live_? It starts failing.

Of course I check logs, but they are empty. Error logs have no errors. 

How could I diagnose this? How could I fix it? It drives me nuts. Worst, I cannot even find any alternatives, new releases. The original repo is stale too. Nginx offers a paid option which costs 9 million dollars. And there are services like Restream and StreamLabs but I bet my kidney they just use the same module / setup somehow because those also fail to work sometimes.

**Comment by cnlohr** [comment]
I've also been having issues like this.  If anyone knows of a way of turning on extremely verbose logging, that would be massively helpful. `docker logs nginx-rtmp` doesn't cut it :-/
